### ISSUE TITLE
Telescope and Microscope subclass of Camera?

### ISSUE THREAD

**Issue by alanruttenberg** [CommonCoreOntology/CommonCoreOntologies#239]
It looks like telescope and microscope both satisfy the definition of camera and so would be subclasses. Currently they are siblings.

**Comment by cameronmore** [comment]
I see how it could be interpreted this way, but the operative parts of the definitions are distinct.

Cameras **record** while microscopes **enable users to see** and telescopes **aid in observation**. I think the emphasis should be read on **record** here, and this could be added as a clarifying annotation.

Unless this is not a convincing interpretation.

**Comment by alanruttenberg** [comment]
The problem is that some telescopes and microscopes are built to record. The telescopes I deal with in my domain are built to record - they are cameras that happen to have telescope lens structure. So there will be multiple inheritance. This needs to be better factored. There is the optics aspect, what kind of lens or mirror is used, and there is a separate aspect which is whether they are used to record or not. All things with lens/mirrors in optical systems share some properties, such as focal length. Imaging sensors have a distinct set of properties (spectral sensitivity, detector size, detector technology).

Off the top of my head, there should probably be a function realized in recording an image, and make camera defined in terms of that function. The spectral range should also be factored out into a relationship so they can be used in defined classes, and those can be re-used for optical filters - there's only Filter in CCO, but I will need to represent optical filters of various sorts. There's another multiple inheritance issue. An optical filter would be both an optical instrument and Filter. Filter is again the sort of class that should be a defined class defined in terms of a function. Also notice that there are parallel subclasses Infrared camera and infrared telescopes, and nothing that links them both as being sensitive to infrared.

By defined class I mean a class that has an equivalentClasses axioms as part of their definition.

**Comment by alanruttenberg** [comment]
Also note the definition of the parent 'imaging instrument': A Material Artifact that is designed to produce images (visual representations) of an entity.

A visual representation is an information entity. If a telescope merely helps you see something by focusing light, that's not producing an representation or image. To be a representation is more than just perceiving what is there.

**Comment by alanruttenberg** [comment]
> But this might not be accurate because the light isn't 'captured' in a simple telescope, but just concentrated so as to let the light hit your eye (or a camera) in a particular way.

Right. Unless you consider the pattern of activation of your vision cells or brain as concretization, which is pushing it. There's also a difference between a screen that has a concretization, like your phone's camera constantly displaying whats in front of it on a screen, vs what happens which you tap to take an image and it is saved to nonvolatile storage. Another example of that is a [ground glass screen on which some telescopes project](https://griffithobservatory.org/exhibits/telescopes/coelostat-solar-telescopes/).

But more importantly this needs to factored better. The focusing element is mix and match with whether there is a sensor making a recording or not. Sensors will have a hierarchy and focusing elements will have a hierarchy and they will be combined using defined classes of artifacts that have both as parts, with a specific function.

**Comment by cameronmore** [comment]
Alan, can we work on refactoring both this and the [optical lenses](https://github.com/CommonCoreOntology/CommonCoreOntologies/issues/230) at the same time? Offline perhaps until we have something concrete?

**Comment by alanruttenberg** [comment]
Yes that makes sense.

**Comment by dlutz2** [comment]
Please add "Periscope" to this set since it is essentially a bent telescope and seems to be misplaced under "Optical Instrument". Also in other CCO usage "Instrument" seems to indicate a complete device where the sub-classes of Optical Instrument (with the exception of "Periscope") would normally be components of a larger assembly or complete device. I believe in earlier CCOs there was a purge of all "X Component" classes. Was the "Optical Instrument" label chosen to avoid the whole-component ambiguity?

**Comment by alanruttenberg** [comment]
Agree on @dlutz2 regarding periscope.

Some more issues related to this.

"Optical" is used in two distinct ways in labels.
"Optical Camera": A Camera that is designed to form and record an image generated from visible light.
"Optical Instrument": A Material Artifact that is designed to process light waves.

I think "optical" should be used the same way, to avoid confusion. Also, what "light" is not defined. I believe the broad view of light is that it is any electromagnetic radiation, and so I would propose clarifying that that's what "light" means. Or, replace light with "electromagnetic radiation". Possibly add electromagnetic radiation, synonym light. In my ontology,  electromagnetic radiation is a process, and its primary label is 'radiating electromagnetic energy" and the definition is: A process during which energy moves through space mediated by the electric field and photon. I have electromagnetic field and photon defined, but that's a larger issue around the definition of energy and related terms.

The proposed minimal changes would be:
- Add a scopeNote to Optical Instrument (and possibly other terms that mention light" clarifying that by "light", any electromagnetic radiation is meant.
- Relabel Optical Camera to be Visible Light Camera.
- Add a subclass of Camera "Optical Camera", with definition "A Camera that is designed to form and record an image generated from light."  (like current "Optical Camera" but without the qualification "visible"
- Make that a Optical Camera be a superclass of Visible Light Camera and Infrared Camera
- Make Video Camera a sibling to Camera, possibly renaming Camera to Still Image Camera. The definition of Camera says "an image" (singular). A Video Camera takes multiple images. 

Separately, add a subclass of Imaging Instrument: Optical System, definition "An aggregate of optical instruments used together to gather and/or manage light". Example: camera optical system, light table with instruments to do photon experiments.
Possibly add a subclass "Camera Optical System": "An Optical System" used by, and which is part of, an Optical Camera. If that's too specific I can define it in my own ontology based on Optical System.

Various fixups due to the changes in definitions, genus-differentia definitions to concord with proposal - My definitions aren't all genus-differentia.

I will contact @cameronmore to try to arrange a meeting soon to work this through.
